requirements.txt

```diff
@@ -1,34 +1,75 @@
-#
-# Requirements for wger for production
-#
+alabaster==0.7.12
+autopep8==1.4.4
+Babel==2.7.0
+bleach==3.1.0
+certifi==2019.3.9
+chardet==3.0.4
+coverage==4.5.3
+defusedxml==0.6.0
 Django==1.9
-django-recaptcha==1.3.1
-reportlab==3.3.0
+django-appconf==1.0.3
+django-bootstrap-breadcrumbs==0.9.1
+django-bower==5.2.0
+django-compressor==2.2
+django-cors-headers==2.4.1
+django-debug-toolbar==1.9
+django-filter==1.1.0
+django-formtools==1.0
 django-mobile==0.7.0
-django-formtools>=1.0,<1.1
-bleach==3.1.0
-python-mimeparse==1.6.0
-Pillow==5.4.1
+django-recaptcha==1.3.1
+django-sortedm2m==1.5.0
+django-tastypie==0.13.3
+djangorestframework==3.2.5
+docutils==0.14
 easy-thumbnails==2.6
-django-compressor==2.2
+entrypoints==0.3
+fitbit==0.3.1
+flake8==3.7.7
+gunicorn==19.9.0
+httplib2==0.13.0
 icalendar==4.0.3
-django-bootstrap-breadcrumbs==0.9.1
-Sphinx==1.8.5
-django-sortedm2m==1.5.0
-django-bower==5.2.0
+idna==2.8
+imagesize==1.1.0
 invoke==1.2.0
-requests==2.21.0
-whitenoise==4.1.2
-gunicorn==19.9.0
+Jinja2==2.10.1
+MarkupSafe==1.1.1
+mccabe==0.6.1
+oauthlib==3.0.1
+packaging==19.0
+pep8==1.7.1
+Pillow==5.4.1
+postgres==2.2.2
 psycopg2==2.7.7
 psycopg2-binary==2.7.7
+pycodestyle==2.5.0
+pyflakes==2.1.1
+Pygments==2.4.2
+PyJWT==1.7.1
+pyparsing==2.4.0
+python-dateutil==2.8.0
+python-mimeparse==1.6.0
+python-oauth2==1.1.0
```

Comment:
@Manorlds-Eaglespark, please make some changes to these requirements, add only the installed requirements than the use of pip refreeze.

Reply:
Thank you @Emmanuel-Dominic  for pointing that out, it has been resolved.